core/src/main/java/io/crate/settings/SharedSettings.java

```diff
@@ -24,14 +24,24 @@
 
 import io.crate.types.DataTypes;
 import org.elasticsearch.common.settings.Setting;
+import org.elasticsearch.common.settings.Settings;
 
 public class SharedSettings {
 
     public static final CrateSetting<Boolean> ENTERPRISE_LICENSE_SETTING = CrateSetting.of(Setting.boolSetting(
-        "license.enterprise", true,
-        Setting.Property.NodeScope), DataTypes.BOOLEAN);
+        "license.enterprise", true, Setting.Property.NodeScope),
+        DataTypes.BOOLEAN);
 
     public static final CrateSetting<String> LICENSE_IDENT_SETTING = CrateSetting.of(Setting.simpleString(
-        "license.ident", Setting.Property.NodeScope, Setting.Property.Dynamic), DataTypes.STRING);
+        "license.ident", Setting.Property.NodeScope, Setting.Property.Dynamic),
+        DataTypes.STRING);
+
+    public static final CrateSetting<Boolean> AUTH_HOST_BASED_ENABLED_SETTING = CrateSetting.of(Setting.boolSetting(
```

Comment:
Could we move these definitions closer to where they're used? Maybe into the Authentication interface? Afaik the SharedSettings class was only created because these enterprise settings are used in multiple modules where we'd otherwise have cyclic dependencies.
Ideally we'd move these into the HostAuthentication, but I guess then we'd need to be able to register additional settings from a Authentication module.

I don't have a strong opinion on this, but I think @seut does